qiita_pet/templates/portals_edit.html

```diff
@@ -0,0 +1,96 @@
+{% extends sitebase.html%}
+
+{%block head%}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript">
+function check_submit(action) {
+  error = false;
+  var data = oTable.$('input').serialize();
```

Comment:
This is absolutely needed no matter what, because of the way datatables interacts with checkboxes. When you get access to the interface, you'll see the error messages added.